src/Components/IssueCard.jsx

```diff
@@ -0,0 +1,362 @@
+import React, {useState, useEffect} from 'react'
+import Paper from '@mui/material/Paper'
+import {makeStyles} from '@mui/styles'
+import Select from '../assets/2D_Icons/Select.svg'
+import Camera from '../assets/2D_Icons/Camera.svg'
+import Share from '../assets/2D_Icons/Share.svg'
+import {TooltipIconButton} from './Buttons'
+import useStore from '../store/useStore'
+import {ISSUE_PREFIX} from './IssuesControl'
+import {addHashParams} from '../utils/location'
+import {setCameraFromEncodedPosition, addCameraUrlParams, removeCameraUrlParams} from './CameraControl'
+import {useIsMobile} from './Hooks'
+
+
+/**
+ * Issue card
+ * @param {string} title The comment body
+ * @param {string} contetne The comment title, optional
+ * @return {Object} React component
+ */
+export default function IssueCard({
```

Comment:
So, it's not clear to me what's required or not here.  Anything that's actually required should have an assertDefined here before continuing to pass values downwards.